src/components/calcite-rating/calcite-rating.e2e.ts

```diff
@@ -4,9 +4,7 @@ import { renders, accessible } from "../../tests/commonTests";
 describe("calcite-rating", () => {
   it("renders", async () => renders("<calcite-rating></calcite-rating>"));
 
-  it("should be accessible", async () => {
-    await accessible(`<calcite-rating></calcite-rating>`);
-  });
+  it("should be accessible", async () => accessible(`<calcite-rating></calcite-rating>`));
```

Comment:
can you enlighten me as to why this one doesn't need an `await` prior to accessible?

Reply:
Because `accessible` returns a promise and can return that directly. The previous way needs it to await on it before resolving the wrapping `async` function. I think most of the common test helpers don't need `async` either because of this.

Reply:
@jcfranco thanks you are a gentleman and a scholar 🎩